test/functional/test_esxi.py

```diff
@@ -0,0 +1,195 @@
+import unittest
+import os
+import time
+import subprocess
+import yaml
+from test import fixtures
+import re
+from infrasim import model
+from infrasim import InfraSimError
+import paramiko
+from infrasim.package_install import install_bintray_packages
+import urllib
+
+old_path = os.environ.get("PATH")
+new_path = "{}/bin:{}".format(os.environ.get("PYTHONPATH"), old_path)
+source_esxi_path = "ftp://10.62.59.23/idic/iso/esxi_img/esxi6p3-1.qcow2"
```

Comment:
what size is this image? It is not appropriate to place it at ftp internally. If small enough, we can place it bintray or even tools repo.